pkg/renew/renew.go

```diff
@@ -211,8 +211,7 @@ func (o *Options) Run(ctx context.Context, args []string) error {
 	}
 
 	for _, crt := range crts {
-		// #nosec G601 -- False positive. See https://github.com/golang/go/discussions/56010
-		if err := o.renewCertificate(ctx, &crt); err != nil {
+		if err := o.renewCertificate(ctx, &crt); /* #nosec G601 -- Pointer does not outlive function scope */ err != nil {
```

Comment:
The `err != nil` is very far to the right and gets hidden easily whenever the text editor isn't wide enough.

Can you change it so that `err != nil` doesn't get pushed too far to the right? For example:

```go
		// #nosec G601 -- Pointer does not outlive function scope
		if err := o.renewCertificate(ctx, &crt); err != nil {
```

Or:

```go
		err = o.renewCertificate(ctx, &crt) /* #nosec G601 -- Pointer does not outlive function scope */
		if err != nil {
```